analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -489,7 +489,11 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
         }
     }
 
-    private fun requestRemoteArtifact(artifact: Artifact, repositories: List<RemoteRepository>): RemoteArtifact {
+    private fun requestRemoteArtifact(
+        artifact: Artifact,
+        repositories: List<RemoteRepository>,
+        useArtifactRepositories: Boolean
```

Comment:
I'm open to better names here. Something like `useReposFromDependencyPoms` sounds a bit clumsy, though.

Reply:
@mnonnenmacher any suggestion here?

Reply:
I actually like the suggestion, maybe a bit clumsy but also precise. Alternatives: `useReposFromDependency`, `useReposDeclaredByDependency`.
I assume you use plural "poms" because repositories from all POMs on the path to the dependency are used?

Reply:
> I assume you use plural "poms" because repositories from all POMs on the path to the dependency are used?

Exactly. So I'll probably go for `useReposFromDependencies` (plural).